tpl/template_test.go

```diff
@@ -276,6 +276,39 @@ func TestIn(t *testing.T) {
 	}
 }
 
+func TestSubstr(t *testing.T) {
+	for i, this := range []struct {
+		v1 string
+		v2 int
+		v3 int
+		expect string
+	}{
+		{"abc", 1, 2, "b"},
+	} {
+		result := Substr(this.v1, this.v2, this.v3)
+
+		if result != this.expect {
+			t.Errorf("[%d] Got %v but expected %v", i, result, this.expect)
+		}
+	}
+}
+
+func TestSplit(t *testing.T) {
+	for i, this := range []struct {
+		v1 string
+		v2 string
+		expect []string
+	}{
+		{"a,b", ",", []string{"a", "b"}},
```

Comment:
You created a table-driven test, so you might as well add another line to this ... maybe another delim?